### PR TITLE
Update about-pull-requests.md

### LINKED ISSUE #430: Clarify which point in each branch is compared in a pull request
Some users are confused about the difference in the commits displayed shown on the compare page and what's shown in a PR.

We talk about what a compare page shows in "[About comparing branches in pull requests](https://help.github.com/articles/about-comparing-branches-in-pull-requests/);" we can add some simple to "[About pull requests](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#about-pull-requests)."

> Compare pages show a compare between the tip of the head ref and the current common ancestor (i.e. merge base) of the head and base ref. On the other hand, pull request pages show a compare between the tip of the head ref and the common ancestor of the head and base ref **at the time when the pull request was created**. In other words, the merge base used for the comparison might be different.

### PR DESCRIPTION
**Pull request by Ruth-ikegah**
### Why:

This PR fixes #430 

### What's being changed:
A section has been added about clarifying the difference in the commits displayed shown on the compare page and what's shown on a pull request page.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by Bocoi501 — APPROVED** [review]

File: content/github/collaborating-with-issues-and-pull-requests/about-pull-requests.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ When you create a pull request, you can choose to create a pull request that is
 
 {% data reusables.pull_requests.mark-ready-review %} You can convert a pull request to a draft at any time. For more information, see "[Changing the stage of a pull request](/articles/changing-the-stage-of-a-pull-request)."
 
+### Difference between commits on Compare page and Pull Request page
```
Comment (Bocoi501) [inline content/github/collaborating-with-issues-and-pull-requests/about-pull-requests.md:56]: ### Further reading

- "[Pull request](/articles/github-glossary/#pull-request)" in the {% data variables.product.prodname_dotcom %} glossary

**Review by Bocoi501 — CHANGES_REQUESTED** [review]

**Comment by janiceilene** [comment]
Thanks @Ruth-ikegah! I'll get this triaged for review ⚡

File: content/github/collaborating-with-issues-and-pull-requests/about-pull-requests.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ When you create a pull request, you can choose to create a pull request that is
...
+
+Compare pages show a compare between the tip of the head ref and the current common ancestor (i.e. merge base) of the head and base ref. On the other hand, pull request pages show a compare between the tip of the head ref and the common ancestor of the head and base ref at the time when the pull request was created. In other words, the merge base used for the comparison might be different.
```
Comment (felicitymay) [inline content/github/collaborating-with-issues-and-pull-requests/about-pull-requests.md:58]: Changes suggested to make this easier for people to read if English isn't their first language.

```suggestion
The compare and pull request pages use different methods to calculate the diff for changed files:

- Compare pages show the diff between the tip of the head ref and the current common ancestor (that is, the merge base) of the head and base ref. 
- Pull request pages show the diff between the tip of the head ref and the common ancestor of the head and base ref at the time when the pull request was created. Consequently, the merge base used for the comparison might be different.
```

**Review by felicitymay — CHANGES_REQUESTED** [review]
👋🏻 Hi @Ruth-ikegah, thanks for adding this information 😄 

It looks as if the issue you were working from was transferred from a really old (2017) internal issue. This was before we started localizing content so the suggested text was out of line with our current recommendations. I've made suggestions to bring it into line.

**Comment by Ruth-ikegah** [comment]
> 👋🏻 Hi @Ruth-ikegah, thanks for adding this information 😄
> 
> It looks as if the issue you were working from was transferred from a really old (2017) internal issue. This was before we started localizing content so the suggested text was out of line with our current recommendations. I've made suggestions to bring it into line.

Thanks for this amazing review @felicitymay I really learnt a lot. I have applied the suggested changes. Thanks.

**Review by felicitymay — APPROVED** [review]
I'm really glad you found the review helpful ✨ 

This looks ready to merge once the tests pass on the updated branch.

I've just noticed from your profile that you're a fellow technical writer. We're a fairly rare species, so it's particularly lovely to meet you 😄

**Comment by Ruth-ikegah** [comment]
> I'm really glad you found the review helpful ✨
> 
> This looks ready to merge once the tests pass on the updated branch.
> 
> I've just noticed from your profile that you're a fellow technical writer. We're a fairly rare species, so it's particularly lovely to meet you 😄

Thanks! It is a pleasure meeting you too. I hope to learn more from you. Is there a community chat for @github docs yet?

**Comment by felicitymay** [comment]
> Is there a community chat for @github docs yet?

That's a great question and I don't know the answer. I'll pass it on to the person who's leading the open source initiative ✨

**Comment by janiceilene** [comment]
👋 @Ruth-ikegah! I'm talking about open sourcing the docs during [Open Source Friday](https://twitter.com/github/status/1316734829367635972) tomorrow! Is that what you're referring to or something else? I'm definitely interested in hearing about what you think would be helpful for folks!

**Comment by Ruth-ikegah** [comment]
> 👋 @Ruth-ikegah! I'm talking about open sourcing the docs during [Open Source Friday](https://twitter.com/github/status/1316734829367635972) tomorrow! Is that what you're referring to or something else? I'm definitely interested in hearing about what you think would be helpful for folks!

Nice I just registered, will be attending. I was referring to something like a community chat channel where contributors of Github docs can network and share ideas.

**Pull request merged**